.github/workflows/ci.yml

```
@@ -40,7 +40,7 @@ jobs:
             ${{ runner.os }}-pnpm-store-
 
       - name: Install dependencies
-        run: pnpm install
+        run: pnpm install --no-frozen-lockfile
```

Why this change? We would like to fail fast if dependencies are stale

---

You're right, I'll fix that.